policyengine_us/variables/household/expense/housing/is_in_public_housing.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class is_in_public_housing(Variable):
+    value_type = bool
+    entity = Household
+    label = "Whether the household is in public housing"
+    definition_period = YEAR
+
+    def formula(household, period, parameters):
+        return (
+            household.sum(
+                household.members.spm_unit("housing_assistance", period)
```

Comment:
housing assistance also includes vouchers. if we don't have another variable for public housing please remove the formula from this one